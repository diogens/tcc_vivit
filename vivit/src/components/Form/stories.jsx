import Form from '.'

export default {
  title: 'Form',
  component: Form
}

export const Basic = (args) => <Form {...args} />