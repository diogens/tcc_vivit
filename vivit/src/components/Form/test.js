import { screen, render } from '@testing-library/react'
import { renderWithTheme } from '../../utils/tests/helpers'

import Form from '.'

describe('<Form/>', () => {
  it('should render the text', () => {
    const { container } = renderWithTheme(<Form />)

    expect(screen.getByText('Form')).toBeInTheDocument()

    expect(container.firstChild).toMatchSnapshot()
  })
})