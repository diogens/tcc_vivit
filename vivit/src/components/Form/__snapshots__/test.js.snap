// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Form/> should render the text 1`] = `
<main
  class="styles__Wrapper-qg3dkt-0"
>
  <h1>
    Form
  </h1>
</main>
`;
